docs/specification/1.0.md

```diff
@@ -92,9 +92,11 @@ curl
    "hookInstance" : "d1577c69-dfbe-44ad-ba6d-3e05e953b2ea",
    "fhirServer" : "http://hooks.smarthealthit.org:9080",
    "hook" : "patient-view",
+   "fhirAuthorization" : "params.tokenUri",
```

Comment:
This should be a JSON object like:

```json
"fhirAuthorization" : {
  "access_token" : "some-opaque-fhir-access-token",
  "token_type" : "bearer",
  "expires_in" : 300,
  "scope" : "patient/Patient.read patient/Observation.read",
  "subject" : "the-cds-service-client-identifier"
}
```

Reply:
@kpshek, agreed.  Although [RFC6750](https://tools.ietf.org/html/rfc6750) discusses "bearer tokens" (using lower case), example code consistently uses upper case.  Thanks for noticing!